src/anonymous-purchase-consequences-screen/AnonymousPurchaseConsequencesScreenComponent.tsx

```diff
@@ -56,18 +59,25 @@ export const AnonymousPurchaseConsequencesScreenComponent = ({
       <ThemeText type="heading--big" color={themeColor} style={styles.header}>
         {t(AnonymousPurchasesTexts.consequences.title)}
       </ThemeText>
-      <Consequence
-        value={t(AnonymousPurchasesTexts.consequences.messages[0])}
-        icon={<ThemeIcon svg={Phone} colorType={themeColor} size="large" />}
-      />
-      <Consequence
-        value={t(AnonymousPurchasesTexts.consequences.messages[1])}
-        icon={<ThemeIcon svg={Receipt} colorType={themeColor} size="large" />}
-      />
-      <Consequence
-        value={t(AnonymousPurchasesTexts.consequences.messages[2])}
-        icon={<ThemeIcon svg={Support} colorType={themeColor} size="large" />}
-      />
+
+      {
+        // eslint-disable-next-line rulesdir/translations-warning
+        AnonymousPurchasesTexts.consequences.messages.map(
+          (consequenceMessage, i) => (
+            <Consequence
+              key={t(consequenceMessage)}
+              value={t(consequenceMessage)}
+              icon={
+                <ThemeIcon
+                  svg={consequencesIcons[i]}
+                  colorType={themeColor}
+                  size="large"
+                />
+              }
+            />
+          ),
+        )
+      }
```

Comment:
I appreciate the drive to get rid of hard coded content, but I feel like this maybe is a bit overkill? Its a bit harder to reason about, plus it breaks if we add more than 3 messages, since there's only 3 items in `consequencesIcons`.

What do you think?

Reply:
Yea might be overkill in this case. I guess we don't expect it to be a long list anyway, so I'll revert it.